projects/ad9656_fmc/common/ad9656_bd.tcl

```diff
@@ -0,0 +1,119 @@
+
+# RX parameters
+set RX_NUM_OF_LANES 4      ; # L
+set RX_NUM_OF_CONVERTERS 4 ; # M
+set RX_SAMPLES_PER_FRAME 1 ; # S
+set RX_SAMPLE_WIDTH 16     ; # N/NP
+
+set RX_SAMPLES_PER_CHANNEL 2 ; # L * 32 / (M * N)
+
+source $ad_hdl_dir/library/jesd204/scripts/jesd204.tcl
+
+# adc peripherals
+
+ad_ip_instance axi_adxcvr axi_ad9656_rx_xcvr
+ad_ip_parameter axi_ad9656_rx_xcvr CONFIG.NUM_OF_LANES $RX_NUM_OF_LANES
+ad_ip_parameter axi_ad9656_rx_xcvr CONFIG.QPLL_ENABLE 1
+ad_ip_parameter axi_ad9656_rx_xcvr CONFIG.TX_OR_RX_N 0
+ad_ip_parameter axi_ad9656_rx_xcvr CONFIG.SYS_CLK_SEL 0
+ad_ip_parameter axi_ad9656_rx_xcvr CONFIG.OUT_CLK_SEL 4
+
+adi_axi_jesd204_rx_create axi_ad9656_rx_jesd $RX_NUM_OF_LANES
+
+ad_ip_instance util_cpack2 util_ad9656_rx_cpack [list \
```

Comment:
We should use this form of instantiation (with a list of parameters) all over the file.

Reply:
Modified the v3 code to use this form of instantiation